<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>672</width>
    <height>221</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">background-color:rgb(0, 191, 255);</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="3">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLineEdit" name="valueEdit">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>40</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Bahnschrift</family>
            <pointsize>13</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>ArrowCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">QLineEdit{
    color:black;
    background-color: rgb(255,255,255);
    border-radius: 5px;
    border: 3px solid red;
}
QLineEdit:hover{
    font-size: 15;
    color:black;
    background-color: rgb(245,245,245);
    border-radius: 5px;
    border: 3px solid rgba(130, 46, 80, 1);
}
</string>
          </property>
          <property name="text">
           <string>0</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="outpuLabel">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>40</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Bahnschrift</family>
            <pointsize>13</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>ArrowCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel{
    font-size: 15;
    color:black;
    background-color: rgb(255,255,255);
    border: 3px solid red;
    border-radius: 5px;
}
QLabel:hover{
   font-size: 15;
    color:black;
    background-color: rgb(245,245,245);
    border: 3px solid rgba(130, 46, 80, 1);
    border-radius: 5px;
}

</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="swapButton">
        <property name="styleSheet">
         <string notr="true">QPushButton {
    float: right;
    margin-right: 3%;
    margin-top: 5px;
    border-radius: 5px;
    padding: 5px 9px;
    font-size: 1.2em;
    background-color: rgba(236, 101, 80, 1);
    border-bottom: 4px solid rgba(217, 91, 72, 1);
    color:white;
    font-family: 'Roboto Slab', serif;
}

QPushButton:hover{
    width: 20px;
    float: right;
    margin-right: 3%;
    margin-top: 10px;
    border-radius: 5px;
    padding: 5px 9px;
    font-size: 1.2em;
    background-color: rgba(130, 46, 80, 1);
    border-bottom: 4px solid rgba(130, 46, 80, 1);
    color:white;
    font-family: 'Roboto Slab', serif;
}
</string>
        </property>
        <property name="text">
         <string>&lt;&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QLabel" name="currentBase">
              <property name="font">
               <font>
                <family>Roboto Slab</family>
                <pointsize>11</pointsize>
                <bold>true</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">color:black;
font-family: 'Roboto Slab', serif;
</string>
              </property>
              <property name="text">
               <string>Из основания:</string>
              </property>
              <property name="margin">
               <number>2</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="toBase">
              <property name="font">
               <font>
                <family>Roboto Slab</family>
                <pointsize>11</pointsize>
                <bold>true</bold>
                <kerning>true</kerning>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">color:black;
font-family: 'Roboto Slab', serif;
</string>
              </property>
              <property name="text">
               <string>В основание:</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QComboBox" name="currentBaseBox">
            <property name="font">
             <font>
              <family>Roboto Slab</family>
              <pointsize>8</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="styleSheet">
             <string notr="true">QComboBox{
    float: right;
    margin-right: 3%;
    margin-top: 5px;
    border-radius: 5px;
    padding: 5px 9px;
    font-size: 1.2em;
    background-color: rgba(236, 101, 80, 1);
    border-bottom: 4px solid rgba(217, 91, 72, 1);
    color:black;
    font-family: 'Roboto Slab', serif;
}
QComboBox:hover{
      float: right;
    margin-right: 3%;
    margin-top: 10px;
    border-radius: 5px;
    padding: 5px 9px;
    font-size: 1.2em;
    background-color: rgba(130, 46, 80, 1);
    border-bottom: 4px solid rgba(130, 46, 80, 1);
    color:white;
   font-family: 'Roboto Slab', serif;
}</string>
            </property>
            <item>
             <property name="text">
              <string>10</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>2</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>3</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>4</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>5</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>6</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>7</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>8</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>9</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>11</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>12</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>13</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>14</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>15</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>16</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>17</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>18</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>19</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>20</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>21</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>22</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>23</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>24</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>25</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>26</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>27</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>28</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>29</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>30</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>31</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>32</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>33</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>34</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>35</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>36</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="toBaseBox">
            <property name="font">
             <font>
              <family>Roboto Slab</family>
              <pointsize>8</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="styleSheet">
             <string notr="true">QComboBox{
    float: right;
    margin-right: 3%;
    margin-top: 5px;
    border-radius: 5px;
    padding: 5px 9px;
    font-size: 1.2em;
    background-color: rgba(236, 101, 80, 1);
    border-bottom: 4px solid rgba(217, 91, 72, 1);
    color:black;
    font-family: 'Roboto Slab', serif;
}
QComboBox:hover{
      float: right;
    margin-right: 3%;
    margin-top: 10px;
    border-radius: 5px;
    padding: 5px 9px;
    font-size: 1.2em;
    background-color: rgba(130, 46, 80, 1);
    border-bottom: 4px solid rgba(130, 46, 80, 1);
    color:white;
   font-family: 'Roboto Slab', serif;
}</string>
            </property>
            <item>
             <property name="text">
              <string>10</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>2</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>3</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>4</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>5</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>6</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>7</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>8</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>9</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>11</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>12</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>13</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>14</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>15</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>16</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>17</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>18</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>19</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>20</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>21</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>22</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>23</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>24</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>25</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>26</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>27</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>28</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>29</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>30</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>31</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>32</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>33</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>34</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>35</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>36</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="exitButton">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>40</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Roboto Slab</family>
        <pointsize>8</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
   float: right;
    margin-right: 3%;
    margin-top: 5px;
    border-radius: 5px;
    padding: 2px 9px;
    font-size: 1.2em;
    background-color: rgba(236, 101, 80, 1);
    border-bottom: 4px solid rgba(217, 91, 72, 1);
    color:white;
   font-family: 'Roboto Slab', serif;
}
QPushButton:hover{
      float: right;
    margin-right: 3%;
    margin-top: 10px;
    border-radius: 5px;
    padding: 2px 9px;
    font-size: 1.2em;
    background-color: rgba(130, 46, 80, 1);
    border-bottom: 4px solid rgba(130, 46, 80, 1);
    color:white;
   font-family: 'Roboto Slab', serif;
}</string>
      </property>
      <property name="text">
       <string>Выход</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Orientation::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>13</width>
        <height>51</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="2">
     <widget class="QPushButton" name="initializeButton">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>40</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Roboto Slab</family>
        <pointsize>8</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
float: right;
margin-right: 3%;
margin-top: 5px;
border-radius: 5px;
padding: 2px 9px;
font-size: 1.2em;
background-color: rgba(236, 101, 80, 1);
border-bottom: 4px solid rgba(217, 91, 72, 1);
color:white;
font-family: 'Roboto Slab', serif;
}
QPushButton:hover{
      float: right;
    margin-right: 3%;
    margin-top: 10px;
    border-radius: 5px;
    padding: 2px 9px;
    font-size: 1.2em;
    background-color: rgba(130, 46, 80, 1);
    border-bottom: 4px solid rgba(130, 46, 80, 1);
    color:white;
   font-family: 'Roboto Slab', serif;
}</string>
      </property>
      <property name="text">
       <string>Очистить</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="copyButton">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>40</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Roboto Slab</family>
        <pointsize>8</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
    float: right;
    margin-right: 3%;
    margin-top: 5px;
    border-radius: 5px;
    padding: 3px 9px;
    font-size: 1.2em;
    background-color: rgba(236, 101, 80, 1);
    border-bottom: 4px solid rgba(217, 91, 72, 1);
    color:white;
    font-family: 'Roboto Slab', serif;
}
QPushButton:hover{
    float: right;
    margin-right: 3%;
    margin-top: 10px;
    border-radius: 5px;
    padding: 3px 9px;
    font-size: 1.2em;
    background-color: rgba(130, 46, 80, 1);
    border-bottom: 4px solidrgba(130, 46, 80, 1);
    color:white;
    font-family: 'Roboto Slab', serif;
}</string>
      </property>
      <property name="text">
       <string>Копировать</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QPushButton" name="convertedButton">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>40</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Roboto Slab</family>
        <pointsize>8</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    float: right;
    margin-right: 3%;
    margin-top: 5px;
    border-radius: 5px;
    padding: 5px 9px;
    font-size: 1.2em;
    background-color: rgba(236, 101, 80, 1);
    border-bottom: 4px solid rgba(217, 91, 72, 1);
    color:white;
    font-family: 'Roboto Slab', serif;
}

QPushButton:hover{
    width: 20px;
    float: right;
    margin-right: 3%;
    margin-top: 10px;
    border-radius: 5px;
    padding: 5px 9px;
    font-size: 1.2em;
    background-color: rgba(130, 46, 80, 1);
    border-bottom: 4px solid rgba(130, 46, 80, 1);
    color:white;
    font-family: 'Roboto Slab', serif;
}
</string>
      </property>
      <property name="text">
       <string>Конвертировать</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>exitButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>240</x>
     <y>378</y>
    </hint>
    <hint type="destinationlabel">
     <x>358</x>
     <y>416</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
